vg/recorder/recorder.go

```diff
@@ -381,29 +385,66 @@ type FillString struct {
 }
 
 // FillString implements the FillString method of the vg.Canvas interface.
-func (c *Canvas) FillString(font vg.Font, x, y vg.Length, str string) {
+func (c *Canvas) FillString(font vg.Font, pt vg.Point, str string) {
 	c.append(&FillString{
-		Font: font.Name(),
-		Size: font.Size,
-		X:    x, Y: y,
+		Font:   font.Name(),
+		Size:   font.Size,
+		Point:  pt,
 		String: str,
 	})
 }
 
 // ApplyTo applies the action to the given vg.Canvas.
 func (a *FillString) ApplyTo(c vg.Canvas) {
-	c.FillString(a.fonts[fontID{name: a.Font, size: a.Size}], a.X, a.Y, a.String)
+	c.FillString(a.fonts[fontID{name: a.Font, size: a.Size}], a.Point, a.String)
 }
 
 // Call returns the pseudo method call that generated the action.
 func (a *FillString) Call() string {
-	return fmt.Sprintf("%sFillString(%q, %v, %v, %v, %q)", a.l, a.Font, a.Size, a.X, a.Y, a.String)
+	return fmt.Sprintf("%sFillString(%q, %v, %v, %v, %q)", a.l, a.Font, a.Size, a.Point.X, a.Point.Y, a.String)
 }
 
 func (a *FillString) callerLocation() *callerLocation {
 	return &a.l
 }
 
+// DrawImage corresponds to the vg.Canvas.DrawImage method
+type DrawImage struct {
+	Rectangle vg.Rectangle
+	Image     image.Image
+
+	l callerLocation
+}
+
+// DrawImage implements the DrawImage method of the vg.Canvas interface.
+func (c *Canvas) DrawImage(rect vg.Rectangle, img image.Image) {
+	c.append(&DrawImage{
+		Rectangle: rect,
+		Image:     img,
+	})
+}
+
+// ApplyTo applies the action to the given vg.Canvas.
+func (a *DrawImage) ApplyTo(c vg.Canvas) {
+	c.DrawImage(a.Rectangle, a.Image)
+}
+
+// Call returns the pseudo method call that generated the action.
+func (a *DrawImage) Call() string {
+	var buf bytes.Buffer
+	err := png.Encode(&buf, a.Image)
+	if err != nil {
+		panic(fmt.Errorf("recorder: error encoding image to PNG: %v", err))
+	}
+	b64 := base64.StdEncoding.EncodeToString(buf.Bytes())
+	str := fmt.Sprintf("{%#v, IMAGE:%s}", a.Image.Bounds(), b64)
+	return fmt.Sprintf("%sDrawImage(%#v, %v)", a.l, a.Rectangle, str)
```

Comment:
`return fmt.Sprintf("%sDrawImage(%#v, {%#v, IMAGE:%s})", a.l, a.Rectangle, a.Image.Bounds(), b64)` and delete L440.